test/e2e/README.md

```diff
@@ -1,14 +1,14 @@
 # Introduction
 
-This directory contain the framework to run complete end-to-end (e2e) tests. It was built upon the [kubernetes-sigs e2e framework](https://github.com/kubernetes-sigs/e2e-framework).
+This directory contain the framework to run complete end-to-end (e2e) tests for single Provider. It was built upon the [kubernetes-sigs e2e framework](https://github.com/kubernetes-sigs/e2e-framework).
 
 # Running end-to-end tests
 
 As long as the cloud provider support is implemented on this framework, you can run the tests
 as shown below for *libvirt*:
 
 ```
-$ BUILTIN_CLOUD_PROVIDERS=libvirt make test-e2e
+$ CLOUD_PROVIDER=libvirt make test-e2e
```

Comment:
We are making the assumption in the test code that we're using `CLOUD_PROVIDER`: https://github.com/confidential-containers/cloud-api-adaptor/blob/841b284803f9a4a8c4a24d7256513ac6cd1f4da3/test/e2e/main_test.go#L27-L31
Of course that could also be changed. Is the idea behind `BUILTIN_CLOUD_PROVIDERS` that we could run the test for multiple cloud providers at a time?

Reply:
For a single cloud provider support my preference would be `CLOUD_PROVIDER` because (as you point out) we are using it elsewhere, but as long as it works I'm sure I can get used to a new approach!

Reply:
Which one should be fine at here, but use `CLOUD_PROVIDER` arg can let use keep same style for `make deploy` and `make image`

Reply:
And the E2E test cases should only can run for one provider in one time, so `CLOUD_PROVIDER` will be better?